integrations/docker/images/stage-1/chip-build-crosscompile/start-sysroot-vm.sh

```diff
@@ -112,11 +112,15 @@ package_upgrade: true
 packages:
   - g++
   - gcc
+  - gstreamer1.0-plugins-base
```

Comment:
just a note: docker will not immediately be updated after this. We will need to build and upload a new sysroot (by we here it is probably me since I got write permissions there) and then we have to update https://github.com/project-chip/connectedhomeip/blob/master/integrations/docker/images/stage-1/chip-build-crosscompile/Dockerfile#L20 to update the `version:` field to the newly build version.